integration-tests/src/test/scala/eu/ostrzyciel/jelly/integration_tests/io/IoSerDesSpec.scala

```diff
@@ -87,6 +87,11 @@ class IoSerDesSpec extends AnyWordSpec, Matchers, ScalaFutures, JenaTest:
     options.maxDatatypeTableSize should be (expOpt.maxDatatypeTableSize)
     options.version should be (Constants.protoVersion_1_0_x)
 
+  private def checkImplOptSupport(impl: NativeSerDes[?, ?], opt: Option[RdfStreamOptions]) =
```

Comment:
I would maybe like a tiny bit of a docstring for this - the conditions you check are self evident, but each of them is in the middle of a longer line of code, so you have to read in to check which conditions the function checks exactly

Reply:
OK, done.